src/main/java/com/faforever/api/utils/FilePermissionUtil.java

```diff
@@ -0,0 +1,28 @@
+package com.faforever.api.utils;
+
+
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.attribute.PosixFilePermission;
+import java.util.HashSet;
+import java.util.Set;
+
+public class FilePermissionUtil {
```

Comment:
* Utility classes should be set `final`
* Utilty classes should have a private constructor